app/assets/stylesheets/application.scss

```diff
@@ -36,66 +55,65 @@ p {
 #description {
   font-size: 175%;
   margin: 50px 1em 50px 1em;
-  padding: 2em 5em 2em 5em;
+  padding: 4em 5em 2em 5em;
```

Comment:
Shorthand form for property `padding` should be written more concisely as `4em 5em 2em` instead of `4em 5em 2em 5em`